Core/MIPS/RiscV/RiscVCompFPU.cpp

```diff
@@ -434,7 +434,7 @@ void RiscVJit::CompIR_FCompare(IRInst inst) {
 		case VC_NN:
 			fpr.MapReg(inst.src1);
 			// NAN is either 0x100 or 0x200.
-			FCLASS(32, SCRATCH1, gpr.R(inst.src1));
```

Comment:
would it be possible to add an assert in the FCLASS emitter to catch this?

Reply:
It did indeed have one - but I guess my `test.py -g` run didn't hit these.  I was pretty sure we had a range of vcmps in pspautotests, so not sure what happened...

Was pretty obvious when the assert tripped in Crisis Core.

-[Unknown]